modules/aws_postgres/tf_module/outputs.tf

```diff
@@ -9,12 +9,12 @@ resource "time_sleep" "wait_for_db" {
 }
 
 output "db_user" {
-  value      = var.existing_global_database_id == null ? aws_rds_cluster.db_cluster[0].master_username : "UNKNOWN_SEE_PRIMARY_DB"
+  value      = (var.existing_global_database_id == null && var.restore_from_snapshot == null) ? aws_rds_cluster.db_cluster[0].master_username : (var.restore_from_snapshot == null ? "UNKNOWN_SEE_PRIMARY_DB" : "UNKNOWN_SEE_ORIGINAL_DB")
   depends_on = [time_sleep.wait_for_db]
 }
 
 output "db_password" {
-  value      = var.existing_global_database_id == null ? aws_rds_cluster.db_cluster[0].master_password : "UNKNOWN_SEE_PRIMARY_DB"
+  value      = (var.existing_global_database_id == null && var.restore_from_snapshot == null) ? aws_rds_cluster.db_cluster[0].master_password : (var.restore_from_snapshot == null ? "UNKNOWN_SEE_PRIMARY_DB" : "UNKNOWN_SEE_ORIGINAL_DB")
```

Comment:
Is this going to cause issues if the DB password is referenced by other modules? Ditto for the DB user and name, but the password is probably the most significant.

Reply:
excellent question
the intent was that if a db was being restored then they would use manual secrets to get the username/db name/password from the old deployment